### PR TITLE
Deprecate AutoMLTrainModelOperator for Vision and Video

### PR DESCRIPTION
**Pull request by VladaZakharova**
---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/system/providers/google/cloud/automl/example_automl_video_intelligence_classification.py [hunk 1]
```diff
@@ -73,73 +72,88 @@
...
+        prediction_type="classification",
+        model_type="CLOUD",
+        dataset_id=video_dataset_id,
+        model_display_name=MODEL_DISPLAY_NAME,
+        region=REGION,
+        project_id=PROJECT_ID,
+    )
+    # [END how_to_cloud_create_video_classification_training_job_operator]
+
+    delete_auto_ml_video_training_job = DeleteAutoMLTrainingJobOperator(
```
Comment (Lee-W) [inline tests/system/providers/google/cloud/automl/example_automl_video_intelligence_classification.py]: Should we make it part of the doc just like `how_to_cloud_create_video_classification_training_job_operator`?

File: tests/system/providers/google/cloud/automl/example_automl_video_intelligence_tracking.py [hunk 2]
```diff
@@ -22,49 +22,49 @@
...
-
-AUTOML_DATASET_BUCKET = f"gs://{DATA_SAMPLE_GCS_BUCKET_NAME}/automl/video_tracking.csv"
-IMPORT_INPUT_CONFIG = {"gcs_source": {"input_uris": [AUTOML_DATASET_BUCKET]}}
-
-
-extract_object_id = CloudAutoMLHook.extract_object_id
+VIDEO_DATA_CONFIG = [
+    {
+        "import_schema_uri": schema.dataset.ioformat.video.object_tracking,
+        # "gcs_source": {"uris": ["gs://cloud-samples-data/ai-platform-unified/video/traffic/traffic_videos_labels.csv"]},
```
Comment (Lee-W) [inline tests/system/providers/google/cloud/automl/example_automl_video_intelligence_tracking.py]: May I know what is this comment for? Should we remove it?

Comment (VladaZakharova) [reply tests/system/providers/google/cloud/automl/example_automl_video_intelligence_tracking.py]: Sure, thank you so much :)

File: tests/system/providers/google/cloud/automl/example_automl_video_intelligence_classification.py (hunk 1 above)
Comment (VladaZakharova) [reply tests/system/providers/google/cloud/automl/example_automl_video_intelligence_classification.py]: Added

File: tests/system/providers/google/cloud/automl/example_automl_nl_text_classification.py [hunk 3]
```diff
@@ -114,7 +112,7 @@
         import_configs=DATA_CONFIG,
     )
 
-    # [START howto_operator_automl_create_model]
+    # [START how_to_cloud_create_text_classification_training_job_operator]
```
Comment (Lee-W) [inline tests/system/providers/google/cloud/automl/example_automl_nl_text_classification.py]: nitpick: Looks like `howto` is the convention of these example DAGs. Not sure whether we want to change it. If so, we probably should change all (might not be in this PR)

**Review by Lee-W — APPROVED** [review]
minor nitpick, but other parts looks good to me

File: tests/system/providers/google/cloud/automl/example_automl_nl_text_classification.py (hunk 3 above)
Comment (VladaZakharova) [reply tests/system/providers/google/cloud/automl/example_automl_nl_text_classification.py]: I have changed this to be more clear what this example is about. Maybe we should change it back, but there are a lot of automl examples for text, translate, video and vision that are named the same way, like `howto_operator_automl_create_model`.
@potiuk What do you think about this?

Comment (potiuk) [reply tests/system/providers/google/cloud/automl/example_automl_nl_text_classification.py]: I think whet @Lee-W proposed here is to just change `how_to` to `howto` (but leave the rest of the change. 

And I think yes - it makes sense for consistency.

Comment (VladaZakharova) [reply tests/system/providers/google/cloud/automl/example_automl_nl_text_classification.py]: Ah, yes, sounds okay for me too :)
Thanks for the clarification! :D

**Comment by VladaZakharova** [comment]
Hi! 
Can you please check this PR? :)
The service will be removed at the end of the month, so will be great to deprecate it at least :) Thank you!

**Review by Lee-W — APPROVED** [review]
Just browsed through it again. LGTM

**Comment by Lee-W** [comment]
If the `text` part is deprecated as well, should we added it to PR title?

**Comment by VladaZakharova** [comment]
> If the `text` part is deprecated as well, should we added it to PR title?

Text was deprecated in another PR, long time ago :)

**Comment by Lee-W** [comment]
> > If the `text` part is deprecated as well, should we added it to PR title?
> 
> Text was deprecated in another PR, long time ago :)

Ah, yes, you're right. Missed the deleted docstring. Thanks for the prompt reply :)

**Comment by VladaZakharova** [comment]
@eladkal 
Hi! Can you please check the PR? Thanks! :)

**Review by potiuk — APPROVED** [review]

**Pull request merged**